lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -515,8 +520,13 @@ def fail_job(self, job_state):
         """
 
         # First we rescue the pods logs
-        with open(job_state.output_file, 'r') as outfile:
-            stdout_content = outfile.read()
+        try:
+            with open(job_state.output_file, 'r') as outfile:
+                stdout_content = outfile.read()
+        except IOError as e:
+            stdout_content = "<Failed to recuperate log>"
```

Comment:
Maybe here indicate the kubernetes job or pod identifier?

Reply:
I mean for the stdout_content...

Reply:
Good thinking @pcm32.  I just pushed [a commit with that addition](https://github.com/ilveroluca/galaxy/commit/cd6c4d4db8bcb72ae931797c216b9829313850cf).  I figure it should appear in this PR as soon as github solves their issues they're experiencing today.